### ISSUE TITLE
[Feature request] Make an endpoint to copy the preprocessing steps from another data frame

### ISSUE THREAD

**Issue by frankcorneliusmartin** [vantage6/vantage6#1568]
This is useful when you've designed an entire workflow after the data extraction job and want to do the same for a different (but similar in variables) extraction job. 

This is useful for both the BlueBerry and IDEA4RC project as they are creating cohorts from the data source but want to apply all preprocessing steps to all data frames.

**Comment by frankcorneliusmartin** [comment]
This is not possible to do at server side as the input of the preprocessing tasks can be encrypted. We could implement that this endpoint only works for un-encrypted collaborations. However since this is an edge case endpoint and therefore I do not want to clutter the code base, I suggest to resolve this client side.

**Comment by bartvanb** [comment]
Updated issue title to reflect this, but it seems better to keep this issue otherwise open to keep this discussion and it's less work than creating a new one and referring to this properly

**Comment by frankcorneliusmartin** [comment]
I actually ment that this need to be implemented this at the IDEA4RC client side (so their RAVEN tool). It could be nice to add this tou 'our' client too. However in that case I feel that we can remove this from the 5.0 release and do this at a later point in time.

If you agree, can you then nock this issue of the #1495 and assign a milestone?

**Comment by bartvanb** [comment]
Yes completely agree. This is a nice-to-have but a great nice-to-have so let's try to do it rather soon but not in the next months / 5.0.
